hack/opm-bundle/bundle.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM scratch
```

Comment:
Do you really want these files duplicated under `hack/opm-bundle`? We don't do this for the other OLM operators. I added a [.gitignore line](https://github.com/openshift/secrets-store-csi-driver-operator/blob/main/.gitignore#L19) for secrets-store (but it's the only one with an explicit ignore).

Reply:
good catch, yes, they should be ignored